lib/test_queue/runner.rb

```diff
@@ -337,6 +349,7 @@ def distribute_queue
               # If we have a slave from a different test run, don't respond, and it will consider the test run done.
               sock.write("OK\n")
               remote_workers += num
+              quorum_members += num
```

Comment:
> My concern with the remote worker count is something connecting 10 times quickly and failing. 

This case is actually hard to detect right now. We only know that a remote worker has stopped when it sends its `WORKER` line over the relay, but if a remote worker crashes or something we don't get any notification. But I think your idea of only tracking active workers is the right one regardless. Maybe someday we'll have a better way of tracking remote worker disconnects.